src/ape/managers/project/manager.py

```diff
@@ -447,16 +447,18 @@ def _get_contract(self, name: str) -> Optional[ContractContainer]:
 
         return None
 
-    # @property
-    # def meta(self) -> PackageMeta:
-    #     return PackageMeta(**self.config_manager.get_config("ethpm").serialize())
+    @property
+    def meta(self) -> PackageMeta:
+        return PackageMeta(**self.config_manager.get_config("ethpm").serialize())
```

Comment:
what happens if this fails to serialize or parse correctly?

Reply:
Agree, please revise to something like this:
```suggestion
        meta = self.config_manager.get_config("meta")
        try:
            return PackageMeta(**meta.serialize())
        except Exception as e:
            raise ConfigError(f"Incorrect configuration of package metadata:\n{meta}") from e
```